intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -66,12 +66,13 @@ Each class definition will have:
 
 - A class name
   - This name is used as the "data type" name
-- Concept-related **attributes**
+- **Attributes**
   - An attribute is a piece of data
   - A class definition holding an attribute means we expect this class to hold a specific piece of data
-- Concept-related **methods**
-  - A method is a function that describes behavior or action
-- Logic to describe what happens every time the class is used to make an **instance.**
+- **Methods**
+  - A **method** is a function that describes behavior or action of a class
+- Logic to describe what happens every time the class is used to make an **instance.** (This is called the `__init__` method. We'll see it in action shortly)
```

Comment:
Do we need to drop the actual Python syntax here? Maybe just refer to it as the initializer or constructor? Putting the full dunder name here feels premature to me especially as the reader has no context on how to read the name and seems likely to get tripped up by the underscores.

I thought that avoiding forward-loading a detail like this was an intentional decision in the original version?

Reply:
I feel that. What if we just treated the constructor as a detail we don't even need to mention here, and let it be introduced in a different lesson?

Reply:
I'm also inclined to make this much less detailed and just say: 
```
Each class definition will have 3 main things:

- A class name
- **State**
- **Behavior**

These are terms we haven't used before but we'll learn more about what **state** and **behavior** mean before the lesson is over.

Reply:
In principle, I'm down with dropping the mention of the constructor. You're right that it's not mentioned elsewhere in this lesson.  If it weren't for Python's strong recommendation that you set up attributes in the ctor (pylint throws a fit if you don't), I'd say yes, and I'm still there like 99% so I guess my vote is to drop it, and to go with the shortened version. 👍